src/code/z_player_lib.c

```diff
@@ -1388,19 +1388,19 @@ f32 sMeleeWeaponLengths[] = {
 Gfx* sBottleDLists[] = { gLinkAdultBottleDL, gLinkChildBottleDL };
 
 Color_RGB8 sBottleColors[] = {
-    { 255, 255, 255 }, // Empty
-    { 80, 80, 255 },   // Fish
-    { 255, 100, 255 }, // Fire
-    { 0, 0, 255 },     // Bug
-    { 255, 0, 255 },   // Poe
-    { 255, 0, 255 },   // Big Poe
-    { 200, 200, 100 }, // Letter
-    { 255, 0, 0 },     // Red Potion
-    { 0, 0, 255 },     // Blue Potion
-    { 0, 255, 0 },     // Green Potion
-    { 255, 255, 255 }, // Milk
-    { 255, 255, 255 }, // Half Milk
-    { 80, 80, 255 },   // Fairy
+    { 255, 255, 255 }, // PLAYER_BOTTLE(PLAYER_IA_BOTTLE)
+    { 80, 80, 255 },   // PLAYER_BOTTLE(PLAYER_IA_BOTTLE_FISH)
+    { 255, 100, 255 }, // PLAYER_BOTTLE(PLAYER_IA_BOTTLE_FIRE)
+    { 0, 0, 255 },     // PLAYER_BOTTLE(PLAYER_IA_BOTTLE_BUG)
+    { 255, 0, 255 },   // PLAYER_BOTTLE(PLAYER_IA_BOTTLE_POE)
+    { 255, 0, 255 },   // PLAYER_BOTTLE(PLAYER_IA_BOTTLE_BIG_POE)
+    { 200, 200, 100 }, // PLAYER_BOTTLE(PLAYER_IA_BOTTLE_RUTOS_LETTER)
+    { 255, 0, 0 },     // PLAYER_BOTTLE(PLAYER_IA_BOTTLE_POTION_RED)
+    { 0, 0, 255 },     // PLAYER_BOTTLE(PLAYER_IA_BOTTLE_POTION_BLUE)
+    { 0, 255, 0 },     // PLAYER_BOTTLE(PLAYER_IA_BOTTLE_POTION_GREEN)
+    { 255, 255, 255 }, // PLAYER_BOTTLE(PLAYER_IA_BOTTLE_MILK_FULL)
+    { 255, 255, 255 }, // PLAYER_BOTTLE(PLAYER_IA_BOTTLE_MILK_HALF)
+    { 80, 80, 255 },   // PLAYER_BOTTLE(PLAYER_IA_BOTTLE_FAIRY)
```

Comment:
personally think these can stay as they were. it ads alot of noise just to get the same information out, and i dont think the fact that its an item action value matters too much in this context

Reply:
I agree but also I'm on a quest to `PLAYER_IA_` shiftability, so commenting where its ordering comes up is useful to me (but if we just don't want that in the repo it's ok)

Reply:
I see. I think maybe a note next to this array (and any other similar scenarios) that says "These are adjusted item action values (`PLAYER_IA_`) " or something that tells you the information and would come up in a search of PLAYER_IA_ would help. But could get some more opinions too